primitives/src/ethereum_checked.rs

```diff
@@ -45,7 +47,7 @@ pub struct CheckedEthereumTx {
     /// Amount to transfer.
     pub value: U256,
     /// Input of a contract call.
-    pub input: BoundedVec<u8, ConstU32<MAX_ETHEREUM_TX_INPUT_SIZE>>,
+    pub input: EthereumTxInput,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The field `input` in the `CheckedEthereumTx` struct has been updated to use the newly introduced `EthereumTxInput` type instead of `BoundedVec<u8, ConstU32<MAX_ETHEREUM_TX_INPUT_SIZE>>`. This change improves code readability and maintains consistency with the new type alias.

LGTM!


<!-- This is an auto-generated comment by OSS CodeRabbit -->